react/ProductDetails.js

```diff
@@ -121,32 +132,42 @@ class ProductDetails extends Component {
               />
             </div>
             <div className="pv2">
-              <hr className="o-30" size="1"/>
+              <hr className="o-30" size="1" />
             </div>
-            {commertialOffer.AvailableQuantity > 0 ?
+            {commertialOffer.AvailableQuantity > 0 ? (
               <div>
                 <div className="pv2">
-                  <BuyButton
-                    seller={sellerId}
-                    skuId={selectedItem.itemId}
-                  >
+                  <BuyButton seller={sellerId} skuId={selectedItem.itemId}>
                     <FormattedMessage id="button-label" />
                   </BuyButton>
-                </div> 
+                </div>
                 <div className="pv4">
-                  <ShippingSimulator skuId={selectedItem.itemId} seller={sellerId} country="BRA" />
+                  <ShippingSimulator
+                    skuId={selectedItem.itemId}
+                    seller={sellerId}
+                    country="BRA"
```

Comment:
This string shouldn't be hardcoded

Reply:
Agree, but I don't see where to get this data